tests/test_other.py

```diff
@@ -9020,9 +9020,9 @@ def test_minimal_runtime_code_size(self):
     test_cases = [
       (asmjs + opts, hello_world_sources, {'a.html': 985, 'a.js': 289, 'a.asm.js': 113, 'a.mem': 6}),
       (opts, hello_world_sources, {'a.html': 972, 'a.js': 624, 'a.wasm': 86}),
-      (asmjs + opts, hello_webgl_sources, {'a.html': 885, 'a.js': 4980, 'a.asm.js': 10972, 'a.mem': 321}),
-      (opts, hello_webgl_sources, {'a.html': 861, 'a.js': 5046, 'a.wasm': 8978}),
-      (opts, hello_webgl2_sources, {'a.html': 861, 'a.js': 6182, 'a.wasm': 8978}) # Compare how WebGL2 sizes stack up with WebGL 1
+      (asmjs + opts, hello_webgl_sources, {'a.html': 885, 'a.js': 5019, 'a.asm.js': 10972, 'a.mem': 321}),
+      (opts, hello_webgl_sources, {'a.html': 861, 'a.js': 5057, 'a.wasm': 8978}),
+      (opts, hello_webgl2_sources, {'a.html': 861, 'a.js': 6191, 'a.wasm': 8978}) # Compare how WebGL2 sizes stack up with WebGL 1
```

Comment:
it's odd that these regressed. these build with ENVIRONMENT=web, so your changes should not have regressed things.

Reply:
Perhaps the extra `{, }` have an effect? Might want to check without that change locally to see. But that's odd - the increase seems larger than that, and closure should be optimizing it anyhow.

Reply:
I submitted those changes before the ifdef changes. I just wanted to see if all the tests passed before reverting them. That should be done now.